src/backend/env/user.rs

```diff
@@ -489,7 +506,7 @@ impl User {
         principals: Vec<String>,
         filter: UserFilter,
         governance: bool,
-        miner: bool,
+        mode: Mode,
```

Comment:
We should do the same on user creation. Currently "Credits" mode is assigned after user creation when frontend sends `update_user`

Reply:
https://github.com/TaggrNetwork/Taggr/commit/aada35055e8114c9157b707b75f1307eb5322edd#r139913008